streams/us.m3u

```diff
@@ -2075,3 +2075,11 @@ https://www.livedoomovie.com/02_AXNHD_720p/chunklist.m3u8
 #EXTINF:-1 tvg-id="" status="error" user-agent="Mozilla/5.0 Macintosh; Intel Mac OS X 10_14_5 AppleWebKit/537.36 KHTML, like Gecko Chrome/76.0.3809.25 Safari/537.36",Warner TV (720p) [Not 24/7]
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 Macintosh; Intel Mac OS X 10_14_5 AppleWebKit/537.36 KHTML, like Gecko Chrome/76.0.3809.25 Safari/537.36
 https://www.livedoomovie.com/02_WarnerTVHD_720p/chunklist.m3u8
+#EXTINF:-1 tvg-id="TropicalMoonCulturaTV.us" status="online",Tropical Moon Cultura TV
+https://srv.panelcast.net/somosculturatv/somosculturatv/playlist.m3u8
```

Comment:
Are you sure this channel is called "Tropical Moon Cultura TV"?

<img width="1433" alt="image" src="https://user-images.githubusercontent.com/7253922/183299743-c8b53148-fd6d-41be-9f3f-0d424cf03960.png">

Reply:
Yes. From [website](https://tropicalmoontv.com/cultura-tv/) the name is Cultura TV, but the logo is Somos Cultura TV. I added the words Tropical Moon to identify its channel of origin because the url also says "cultura". I can rename it to "Somos Cultura" because of its logo, do you agree?

Reply:
For now I will accept this PR, to create another one and focus on its name change (because it would change its entry of database repo).